.env.vagrant

```diff
@@ -34,7 +31,9 @@ OAUTH2_TOKEN_URI=http://web.chs-dev.internal:4000/oauth2/token
 OAUTH2_CLIENT_ID=
 OAUTH2_CLIENT_SECRET=
 
+FILE_TRANSFER_FEATURE_ENABLED=1
```

Comment:
Do not forget to remove it when refactor is merged in. I suspect there might be code conflict around this place.